src/Transaction/TransactionException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace PhpInPractice\Matters\Aggregate\Transaction;
+
+class TransactionException extends \RuntimeException
+{
+
+}
```

Comment:
I don't really like code that doesn't serve a concrete use case, but I'm 2/10 on this, so if you really want to keep this in I'm fine with that